tests/conftest.py

```diff
@@ -392,3 +370,20 @@ def random_dt(monkeypatch):
     """Make ``datetime.datetime.utcnow()`` return an arbitrary date."""
     with mocked_datetime(monkeypatch) as datetime:
         yield datetime.utcnow()
+
+
+@pytest.fixture(scope="function")
+def setup_pickleddb_database():
+    """Configure the database"""
+    with OrionState() as cfg:
```

Comment:
It this still relying on a tmp file for the DB?

Reply:
yes

Reply:
`OrionState.make_config`